diff -rc owncloud_old/core/setup/controller.php owncloud/core/setup/controller.php
*** owncloud_old/core/setup/controller.php	2016-01-19 19:14:41.833956051 +0000
--- owncloud/core/setup/controller.php	2016-01-19 19:17:00.673954157 +0000
***************
*** 39,45 ****
  	 * @param Setup $setupHelper
  	 */
  	function __construct(Setup $setupHelper) {
! 		$this->autoConfigFile = \OC::$SERVERROOT.'/config/autoconfig.php';
  		$this->setupHelper = $setupHelper;
  	}
  
--- 39,50 ----
  	 * @param Setup $setupHelper
  	 */
  	function __construct(Setup $setupHelper) {
! 		$config_directory = getenv('OWNCLOUD_CONFIG_DIR');
! 		if ($config_directory) {
! 			$this->autoConfigFile = $config_directory.'/autoconfig.php';
! 		} else {
! 			$this->autoConfigFile = \OC::$SERVERROOT.'/config/autoconfig.php';
! 		}
  		$this->setupHelper = $setupHelper;
  	}
  
diff -rc owncloud_old/lib/base.php owncloud/lib/base.php
*** owncloud_old/lib/base.php	2016-01-19 19:14:42.293956045 +0000
--- owncloud/lib/base.php	2016-01-19 19:14:46.037955994 +0000
***************
*** 128,134 ****
  		} elseif(defined('PHPUNIT_RUN') and PHPUNIT_RUN and is_dir(OC::$SERVERROOT . '/tests/config/')) {
  			self::$configDir = OC::$SERVERROOT . '/tests/config/';
  		} else {
! 			self::$configDir = OC::$SERVERROOT . '/config/';
  		}
  		OC_Config::$object = new \OC\Config(self::$configDir);
  
--- 128,139 ----
  		} elseif(defined('PHPUNIT_RUN') and PHPUNIT_RUN and is_dir(OC::$SERVERROOT . '/tests/config/')) {
  			self::$configDir = OC::$SERVERROOT . '/tests/config/';
  		} else {
! 			$config_directory = getenv('OWNCLOUD_CONFIG_DIR');
! 			if($config_directory) {
! 				self::$configDir = $config_directory.'/';
! 			} else {
! 				self::$configDir = OC::$SERVERROOT . '/config/';
! 			}
  		}
  		OC_Config::$object = new \OC\Config(self::$configDir);
