diff -rc mysql-5.7.10_old/storage/innobase/buf/buf0flu.cc mysql-5.7.10/storage/innobase/buf/buf0flu.cc
*** mysql-5.7.10_old/storage/innobase/buf/buf0flu.cc	2016-01-15 22:54:37.772838988 +0000
--- mysql-5.7.10/storage/innobase/buf/buf0flu.cc	2016-01-15 22:55:09.640838554 +0000
***************
*** 2962,2969 ****
  buf_flush_page_cleaner_set_priority(
  	int	priority)
  {
! 	setpriority(PRIO_PROCESS, (pid_t)syscall(SYS_gettid),
! 		    priority);
  	return(getpriority(PRIO_PROCESS, (pid_t)syscall(SYS_gettid))
  	       == priority);
  }
--- 2962,2970 ----
  buf_flush_page_cleaner_set_priority(
  	int	priority)
  {
!  	// setpriority isn't allowed on Ubuntu Core.
! 	//setpriority(PRIO_PROCESS, (pid_t)syscall(SYS_gettid),
! 	//	    priority);
  	return(getpriority(PRIO_PROCESS, (pid_t)syscall(SYS_gettid))
  	       == priority);
  }
